lib/actions/config.js

```diff
@@ -1,3 +1,6 @@
 import { createAction } from 'redux-actions'
 
 export const setAutoPlan = createAction('SET_AUTOPLAN')
+
+// TODO: this should eventually be handled via mapState
+export const setMapCenter = createAction('SET_MAP_CENTER')
```

Comment:
Is this used anywhere else in the app?  If it's just a trimet-mod-otp thing, it should probably live in that project.